tests/framework/cli/test_starters.py

```diff
@@ -1311,7 +1312,8 @@ def test_invalid_example(self, fake_kedro_cli, bad_input):
 
         assert result.exit_code != 0
         assert (
-            "It must contain only y, n, YES, NO, case insensitive.\n" in result.output
+            "It must contain only y, n, YES, or NO (case insensitive).\n"
```

Comment:
Should we test for the full expected error message here? Like we do for the test above

```
assert f"{bad_input} is an invalid value for example pipeline." in result.output`
```

Reply:
Done!